std_msgs/Header header
uint8 num_objects
uint16[10] center_x
uint16[10] center_y
float32[10] distance